ProfessorAssessor/Networking/Networker.h

```diff
@@ -24,9 +25,22 @@ NS_ASSUME_NONNULL_BEGIN
          withRating:(NSNumber *)rating
      withCompletion:(PFBooleanResultBlock)completion;
 
-// TODO: add method for fetching all reviews for a professor
-
-// TODO: add method for fetching reviews for a professor for a specific course
++ (void)fetchSchoolAndProfessorsWithCompletion:(
+                                                void(^)
+                                                (PFObject *_Nullable object,
+                                                 NSError *_Nullable error))completion;
+
++ (void)fetchReviewsForProfessor:(Professor *)professor
+                      forCourses:(NSArray<Course *> *)courses
+                  withCompletion:(
+                                  void(^)
+                                  (NSArray<Review *> *_Nullable objects,
+                                   NSError *_Nullable error))completion;
```

Comment:
This is a funny way of formatting blocks but it's consistent so I'm totally fine with it.